elk-setting/docker-compose.yml

```
@@ -0,0 +1,208 @@
+      [ ! -f /var/lib/logstash/metadata/last_run_metadata_franchises_posts.yml ] && touch /var/lib/logstash/metadata/last_run_metadata_franchises_posts.yml && \
+      logstash -f /usr/share/logstash/pipeline/logstash.conf"
+    networks:
+      - elastic-network
+
+
+networks:  # 네트워크 설정
+  elastic-network:  # elastic-network 이름 명시
+    external: false  # 외부 네트워크를 사용하지 않음
+
+
+# # networks 항목에 default를 정의하면, 도커 컴포즈는 기본적으로 모든 서비스 컨테이너를 이 네트워크에 자동으로 연결
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)